src/tests/ftest/soak/soak.py

```diff
@@ -872,16 +894,15 @@ def run_soak(self, test_param):
         self.all_failed_jobs = []
         # cleanup soak log directories before test on all nodes
         result = slurm_utils.srun(
-            NodeSet.fromlist(self.hostlist_clients), "rm -rf {}".format(
-                self.log_dir), self.srun_params)
+            NodeSet.fromlist(self.hostlist_clients),
+            ["rm", "-rf", self.log_dir], self.srun_params)
         if result.exit_status > 0:
             raise SoakTestError(
                 "<<FAILED: Soak directories not removed"
                 "from clients>>: {}".format(self.hostlist_clients))
         # cleanup test_node /tmp/soak
-        cmd = "rm -rf {}".format(self.log_dir)
         try:
-            result = process.run(cmd, shell=True, timeout=30)
+            result = process.run(["rm", "-rf", self.log_dir], timeout=30)
```

Comment:
Do we need a full path for the `rm` command?

Reply:
not needed for rm

Reply:
not needed.